indexing-service/src/main/java/org/apache/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -236,7 +236,7 @@ public TaskStatus call()
 
                         if (node.isEnablePlaintextPort()) {
                           if (config.isSeparateIngestionEndpoint()) {
-                            Pair<Integer, Integer> portPair = portFinder.findTwoConsecutiveUnusedPorts();
+                            Pair<Integer, Integer> portPair = portFinder.findTwoUnusedPorts();
```

Comment:
I think this will break anyone who really used the `isSeparateIngestionEndpoint()` feature. If I recall correctly then tranquility assumes ingestion port to be `peonPort + 1` and that would break if the ports are not consecutive.
that said, I am not 100% sure if above is true and if we care about it anymore.
@gianm ?

Reply:
maybe we should remove that config and let users rely on QoS filter only.

Reply:
@himanshug Now I know why we have to generate two consecutive unused ports after have read your comments. I have never used the `isSeparateIngestionEndpoint()` feature before. If we want to support `druid.indexer.runner.ports` feature, it's hard to guarantee ports is consecutive unless we asked users to provide consecutive ports, which is unfriendly to users.

Reply:
I agree with you to remove `separateIngestionEndpoint` config for it's not supported with TLS.

Reply:
@himanshug @gianm Can we remove  `separateIngestionEndpoint` config, any conclusion?